assets/js/components/Dashboard/AlertsPage.tsx

```diff
@@ -37,28 +38,21 @@ interface AlertsResponse {
   screens_by_alert: ScreensByAlert;
 }
 
-interface Props {
-  places: Place[];
-  isVisible: boolean;
-}
-
-const AlertsPage: ComponentType<Props> = (props: Props) => {
-  const { places, isVisible } = props;
+const AlertsPage: ComponentType = () => {
+  const { places } = useOutletContext<{ places: Place[] }>();
```

Comment:
I did try that first, and it does work great. Only issue was tests. Using a custom hook from `Dashboard` means that the `render` function in each test would need to reference `Dashboard`. My first instinct was that isn't ideal, but it may be ok for the reference to it to live in the util Component I made for routing. What do you think?